R/fftrees_create.R

```diff
@@ -224,164 +232,175 @@ fftrees_create <- function(data = NULL,
   }
 
   testthat::expect_true(!is.null(cost.outcomes),
-    info = "cost.outcomes is NULL"
+                        info = "cost.outcomes is NULL"
   )
 
   testthat::expect_is(cost.outcomes,
-    class = "list",
-    info = "cost.outcomes must be a list in the form list(hi = x, mi = x, fa = x, cr = x)"
+                      class = "list",
+                      info = "cost.outcomes must be a list in the form list(hi = x, mi = x, fa = x, cr = x)"
   )
 
   testthat::expect_true(all(names(cost.outcomes) %in% c("hi", "mi", "fa", "cr")),
-    info = "cost.outcomes must be a list in the form list(hi = x, mi = x, fa = x, cr = x)"
+                        info = "cost.outcomes must be a list in the form list(hi = x, mi = x, fa = x, cr = x)"
   )
 
-  ## cost.cues: ----
+
+  # cost.cues: ----
 
   if (!is.null(cost.cues) & goal != "cost") {
-    message("Note: You specified cost.cues but goal = '", goal, "' (not 'cost'). Trees will ignore these costs during growth")
+    message("Note: You specified cost.cues but goal = '", goal, "' (not 'cost'). Trees will ignore these costs during growth.")
   }
 
 
-  # Append cost.cues
+  # Append cost.cues:
   cost.cues <- cost.cues.append(formula,
-    data,
-    cost.cues = cost.cues
+                                data,
+                                cost.cues = cost.cues
   )
 
   testthat::expect_true(!is.null(cost.cues),
-    info = "cost.cues is NULL"
+                        info = "cost.cues is NULL"
   )
 
   testthat::expect_is(cost.cues,
-    class = "list"
+                      class = "list"
   )
 
   testthat::expect_true(all(names(cost.cues) %in% names(data)),
-    info = "At least one of the values you specified in cost.cues is not in data"
+                        info = "At least one of the values specified in cost.cues is not in data"
   )
 
-  ## stopping.rule: ----
+  # stopping.rule: ----
 
   stopping.rule_valid <- c("exemplars", "levels")
 
   testthat::expect_true(stopping.rule %in% stopping.rule_valid)
 
 
-  ## stopping.par: ----
+  # stopping.par: ----
 
   testthat::expect_gt(stopping.par, expected = 0)
   testthat::expect_lt(stopping.par, expected = 1)
 
-  ## decision.labels: ----
+
+  # decision.labels: ----
 
   testthat::expect_true(!is.null(decision.labels),
-    info = "decision.labels is NULL"
+                        info = "decision.labels is NULL"
   )
 
   testthat::expect_equal(length(decision.labels), 2)
 
-  ## repeat.cues: ----
+
+  # repeat.cues: ----
   testthat::expect_is(repeat.cues, "logical")
 
 
-  # Data quality checks: ------
 
-  ## Criterion is in data: ----
+  # 2. Data quality checks: ------
+
+  # Criterion is in data: ----
 
   testthat::expect_true(criterion_name %in% names(data),
-    info = paste("The criterion", criterion_name, "is not in your data object")
+                        info = paste("The criterion", criterion_name, "is not in data")
```

Comment:
@hneth I am seeing a lot of spacing changes. Did you add these manually or were they created from an automatic styler?

If manually, I'd prefer we stick with using an automatic code styler like `styler::style_pkg()`. 

If automatically, which did you use?

Reply:
Thanks for the timely review of all these changes, as well as careful & detailed comments!  I'll respond to some general points below, after commenting on the details here. 

The styling has been manual so far.  I'm not familiar with any automatic stylers, but would not mind trying them out in a future revision. At this point, however, I'd prefer prioritizing issues of functionality.

(My changes of styling so far only served to structure the code so that I can track and understand it, e.g., by adding comments and section markers.  But I realize that other users may have other needs, so it's good to adopt a standard style eventually.)

Reply:
Makes sense. Indeed code styling can be a personal preference! Personally I use the `styler::style_pkg()` function in https://www.tidyverse.org/blog/2017/12/styler-1.0.0/, which will take care of a lot of (but not all) non-functional styling.

I'm ok to leave all styling changes in this PR, but would request that future styling changes be kept in separate PRs. I'm just afraid that with lots of styling changes mixed with functional changes, I'll fail to detect critical changes that need review.